requirements.txt

```diff
@@ -41,3 +41,4 @@ python-socketio==3.1.1
 pydot==1.4.1
 keras-applications==1.0.6
 keras-preprocessing==1.0.5
+rasa~=0.2.0a3
```

Comment:
this will create a cyclic dependency - are you sure the installation still works? 😅  might be better to just assume it is installed and error if the package can not be found.

Reply:
but not too important as it will be fixed by core & nlu merge but still want to make sure install still works.

Reply:
If helpful, we can also already merge the rasa_stack code in the core repository

Reply:
yes, I thought of this as an intermediary solution until the repos are merged (and yes installation works - I needed this for the tests)

Reply:
:+1:

Reply:
now that the repositories are merged, let's remove this requirement :tada: